modules/recommended-node-host-practices.adoc

```diff
@@ -46,3 +46,9 @@ of the properties of the node.
  kubeletConfig:
     maxPods: 250
 ----
+
+The rate at which the kubelet talks to the API server depends on qps and burst
+values. The default values, `20` for `kube-api-qps` and `40` for
```

Comment:
@ahardin-rh the default values of kube-api-qps and kube-api-burst rates are 5/10 respectively for OCP 4.x, do you think it's a good idea to add the procedure or maybe mention in the https://docs.openshift.com/container-platform/4.1/scalability_and_performance/recommended-host-practices.html#create-a-kubeletconfig-crd-to-edit-kubelet-parameters_ section the params to tweak to change the values?

Reply:
They got bumped to 50/100 recently but I think the change will get in the 4.2.z builds - https://github.com/openshift/machine-config-operator/pull/1173, here is how we can tweak the kubelet qps and burst rates :
```
apiVersion: machineconfiguration.openshift.io/v1
kind: KubeletConfig
metadata:
  name: set-max-pods
spec:
  machineConfigPoolSelector:
  kubeletConfig:
    maxPods: <pod_count>
    kubeAPIBurst: <burst_rate>
    kubeAPIQPS: <qps>
```

Reply:
@chaitanyaenr Thank you! I made some updates. PTAL: